tools/Utilities/src/ViewModels/UtilityViewModel.cs

```diff
@@ -18,6 +19,8 @@ public class UtilityViewModel : INotifyPropertyChanged
 
     private readonly string exeName;
 
+    private bool launchAsAdmin;

```

Comment:
Can this just be an [ObservableProperty] so that we don't need to public bool below?
Also, generally use the _ prefix on private class variables.

Reply:
> Also, generally use the _ prefix on private class variables.
++1

Reply:
However, if `launchAsAdmin` is not controlling the UI, no need to add the ObservableProperty attribute.